lib/form/paragraph3.flow

```diff
@@ -1511,15 +1522,17 @@ optimizeLine(words : [ParaWord], alignment: ParaLineAlignment, availableWidth :
 					Text(t1, s1) : {
 						switch (f : GhostForm) {
 							Text(t2, s2) : {
+								s1_withoutLS = if (ignoreLetterSpacing) removeAllStructs(s1, LetterSpacing(.0)) else s1; 
```

Comment:
s1_withoutLS and s2_withoutLS are confusing because they are with letterspacng if !ignoreLetterSpacing